components/user_profile/user_profile.jsx

```diff
@@ -22,6 +22,7 @@ export default class UserProfile extends PureComponent {
         overwriteName: PropTypes.node,
         user: PropTypes.object,
         userId: PropTypes.string,
+        showGuestBadge: PropTypes.bool,
```

Comment:
What does this have to do with the current issue?

Reply:
@Willyfrog the User profile component by default has the `Guest` label/tag. So to avoid it in the post comment, this change was brought in. 
(if this tag should be carried forward, we have to play with css and we can remove this piece of code.)

![Screenshot from 2020-03-30 20-20-12](https://user-images.githubusercontent.com/17671710/77926612-ef778c80-72c3-11ea-87d8-0681cbc9943e.png)

Reply:
but it should only have the tag if the account is a guest one, right? Taking a look at the jira ticket, I don't see any notes asking for removal.

Reply:
Thanks @Willyfrog. I'll update it soon.

Reply:
Addressed in latest commit.